data/mapcss-mapping.csv

```diff
@@ -1062,8 +1062,12 @@ highway|speed_camera;991;
 shop|beauty;992;
 shop|sports;993;
 route|ferry|motor_vehicle;[route=ferry][motor_vehicle];x;name;int_name;994;route|ferry
-# For railways that carry vehicles, used for routing similar to ferries:
-railway|rail|motor_vehicle;[railway=rail][motor_vehicle];;name;int_name;995;
+# For railways that carry vehicles, used for routing similar to ferries.
+# TODO: deprecated June 2023 in favor of non-drawable railway-motor_vehicle
+# (because we don't want to have two drawable types in one feature,
+# e.g. railway-rail-motor_vehicle and railway-rail-main|branch|etc)
+# but should be retained for some time to be drawable in old mwms.
+railway|rail|motor_vehicle;[railway=rail][dont=match];;name;int_name;995;
```

Comment:
It can (should) be replaced with ;railway|rail|main

Reply:
Also routing code should be updated: RailwayRailMotorVehicle = 994
BTW, it has an error now ...

Reply:
so you mean abolish the `railway-rail-motor_vehicle` type and associated special routing code?

Reply:
This railway will be drawn like main or branch. and you can delete it from styles.
Routing will be broken for old MWMs anyway, but this is not a big problem.

Reply:
routing logic should be modified on new type

Reply:
I believe I did it already in https://github.com/organicmaps/organicmaps/pull/5390/commits/8dbf61b94eb82fde08e347cbdc0c36ecf3967471

Reply:
Put here:
railway|rail|motor_vehicle;995;railway|rail